locales/cy/dashboard.json

```
@@ -1,34 +1,34 @@
 {
-    "add_users_if_they_need_to_use_services_for": "[CY] add users if they need to use services for the authorised agent",
-    "anti_money_laundering_registration_details_for": "[CY] Anti-Money Laundering (AML) registration details for this business",
-    "authorised_agent_number": "[CY] Authorised agent number:",
-    "authorised_agent_services": "[CY] Authorised agent services",
-    "authorised_agent_status": "[CY] Authorised agent status:",
-    "authorised_agents_are_also_known_as": "[CY] Authorised agents are also known as Authorised Corporate Service Providers (ACSPs).",
-    "business_details": "[CY] business details",
-    "close_the_authorised_agent": "[CY] Close the authorised agent",
+    "add_users_if_they_need_to_use_services_for": "ychwanegu defnyddwyr newydd os oes angen iddynt ddefnyddio gwasanaethau ar gyfer yr asiant awdurdodedig",
+    "anti_money_laundering_registration_details_for": "manylion cofrestru Atal Gwyngalchu Arian (AML) ar gyfer y busnes hwn",
+    "authorised_agent_number": "Rhif asiant awdurdodedig:",
+    "authorised_agent_services": "Gwasanaethau asiant awdurdodedig",
+    "authorised_agent_status": "Statws asiant awdurdodedig:",
+    "authorised_agents_are_also_known_as": "Gelwir asiantau awdurdodedig hefyd yn Ddarparwyr Gwasanaeth Corfforaethol Awdurdodedig (ACSP).",
```

Not sure how we should be handling Welsh acronyms, might be worth raising on the ticket along with the rest of the missing translations?

```suggestion
    "authorised_agents_are_also_known_as": "Gelwir asiantau awdurdodedig hefyd yn Ddarparwyr Gwasanaeth Corfforaethol Awdurdodedig (DGCAs).",
```

---

Provided Welsh translations use 'ACSP' too, so will not edit for the time being